prac_05/state_names.py

```diff
@@ -0,0 +1,23 @@
+# Adrianto Hartono
+# 14347930
+
+"""
+ CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+# TODO: Reformat this file so the dictionary code follows PEP 8 convention
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+
+for code in CODE_TO_NAME:
+    print(f"{code:3} is {CODE_TO_NAME[code]}")
+
+state_code = input("Enter short state: ").upper()
+while state_code != "":
+    if state_code in CODE_TO_NAME:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+    else:
+        print("Invalid short state")
+    state_code = input("Enter short state: ")
```

Comment:
Great code it works!

Reply:
I think that you should add .upper() in 23 line

Reply:
Easy to understand and it works!

Reply:
thank you for the feedback :)